exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/StringFunctions.java

```diff
@@ -770,6 +770,65 @@ public void eval() {
     } // end of eval
   }
 
+  /*
+   * Fill up the string to length 'length' by prepending the character ' ' in the beginning of 'text'.
+   * If the string is already longer than length, then it is truncated (on the right).
+   */
+  @FunctionTemplate(name = "lpad", scope = FunctionScope.SIMPLE, nulls = NullHandling.NULL_IF_NULL)
+  public static class LpadTwoArg implements DrillSimpleFunc {
```

Comment:
I already have some in TestFunctionsWithTypeExpoQueries
I added more to TestStringFunctions